apps/content/repositories/mushaf.py

```
@@ -0,0 +1,166 @@
+        file: Any = None,
+    ) -> AssetVersion:
+        """
+        Create an AssetVersion for the asset.
+        """
+        with transaction.atomic():
+            size_bytes = 0
+            if file:
+                try:
+                    size_bytes = file.size
+                except Exception:
+                    pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)